src/modules/avatar/controller/inputController.ts

```diff
@@ -510,8 +545,10 @@ export class InputController extends ScriptComponent {
                 // Allow the avatar to move in the air.
                 this._doMoveInJumping(delta);
                 // Move to the nxt jump substate once the avatar touches the ground.
-                if (this._detectGround()
-                    || Math.abs(this._avatarState.previousPosY - this._gameObject.position.y) < 0.001) {
+                if (
+                    this._detectGround()
+                    || Math.abs(this._avatarState.previousPosY - this._gameObject.position.y) < 0.001
+                ) {
```

Comment:
Looks a bit weird but if the linter lets it past, OK.

Reply:
Good spotting. This statement exceeded the line length limit at some point in the past. I didn't think to check if it was still too long before reformatting it.
Updated with commit [c245b5b](https://github.com/vircadia/vircadia-web/pull/109/commits/c245b5b858911508e6c81a1d0d1d8ba448b9d9b9), thanks!